acme/dns_challenge.go

```diff
@@ -31,6 +31,10 @@ var defaultNameservers = []string{
 
 var RecursiveNameservers = getNameservers(defaultResolvConf, defaultNameservers)
 
+// DNSPropagationTimeout is used to override the DNS propagation timeout (if left unset, this timeout is inherited from
+// the DNS provider if implemented, or defaults to 60 seconds if not).
+var DNSPropagationTimeout = 0 * time.Second
```

Comment:
Can just be:

```
var DNSPropagationTimeout time.Duration
```

And simplify the comment; it makes claims that aren't about the variable being defined, and which will easily become wrong if timeout values are changed ~100 lines down.